src/content/docs/plugin/notification.mdx

```diff
@@ -152,21 +147,162 @@ tauri::Builder::default()
 
 ### Actions
 
-{/* TODO: */}
+Actions add interactive buttons and inputs to notifications. Use them to create a responsive experience for your users.
+
+#### Register Action Types
+
+Register action types to define interactive elements:
+
+```javascript
+import { registerActionTypes } from '@tauri-apps/plugin-notification';
+
+await registerActionTypes([
+  {
+    id: 'messages',
+    actions: [
+      {
+        id: 'reply',
+        title: 'Reply',
+        input: true,
+        inputButtonTitle: 'Send',
+        inputPlaceholder: 'Type your reply...',
+      },
+      {
+        id: 'mark-read',
+        title: 'Mark as Read',
+        foreground: false,
+      },
+    ],
+  },
+]);
+```
+
+#### Action Properties
+
+| Property                 | Description                             |
+| ------------------------ | --------------------------------------- |
+| `id`                     | Unique identifier for the action        |
+| `title`                  | Display text for the action button      |
+| `requiresAuthentication` | Requires device authentication          |
+| `foreground`             | Brings app to foreground when triggered |
+| `destructive`            | Shows action in red on iOS              |
+| `input`                  | Enables text input                      |
+| `inputButtonTitle`       | Text for input submit button            |
+| `inputPlaceholder`       | Placeholder text for input field        |
+
+#### Listen for Actions
 
-<Stub />
+Listen to user interactions with notification actions:
+
+```javascript
+import { onAction } from '@tauri-apps/plugin-notification';
+
+await onAction((notification) => {
+  console.log('Action performed:', notification);
+});
+```
 
 ### Attachments
 
-{/* TODO: */}
+Attachments add media content to notifications. Support varies by platform.
+
+```javascript
+import { sendNotification } from '@tauri-apps/plugin-notification';
+
+sendNotification({
+  title: 'New Image',
+  body: 'Check out this picture',
+  attachments: [
+    {
+      id: 'image-1',
+      url: 'asset:///notification-image.jpg',
```

Comment:
just checking because I don't know, is asset triple slashed? Any reference to it?

Reply:
i am also not sure, cause it supports both protocol, and it depends on how it's configured 
https://github.com/tauri-apps/plugins-workspace/blob/0ca4cc914c5ea995c98f9e60a2ab49827c219350/plugins/notification/guest-js/index.ts#L231
```
  /** Attachment URL. Accepts the `asset` and `file` protocols. */
  url: string
  ```